gh-badges/package.json

```diff
@@ -36,7 +36,7 @@
   "dependencies": {
     "anafanafo": "^1.0.0",
     "dot": "^1.1.2",
-    "gm": "^1.23.0",
+    "gm": "^1.23.1",
```

Comment:
Thought updating this file might help (it did not). I can roll this back if anyone has concerns, but I don't see any issues with the patch bump on this dependency